app/components/InputText.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+    input::-webkit-input-placeholder {
+        color: ${props => props.placeholderColor} !important;
```

Comment:
Just added them because input is a non-specific selector that could be overridden easily